cockatrice/src/carddatabase.cpp

```diff
@@ -298,6 +300,10 @@ static QXmlStreamWriter &operator<<(QXmlStreamWriter &xml, const CardInfo *info)
     for (int i = 0; i < related.size(); i++)
         xml.writeTextElement("related", related[i]);
 
+    const QStringList &reverseRelated = info->getReverseRelatedCards();
+    for (int i = 0; i < reverseRelated.size(); i++)
```

Comment:
Yes, ```foreach``` was available even before c++11 thanks to Qt.
Anyway, since the code was already using index/based looping in the xml writer, i've kept the newly added code coherent with the older.